retail/interactive-tutorial/RetailEvents.Samples/PurgeUserEventSample.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_purge_user_event]
+// Import user events into a catalog from inline source using Retail API
+
+using Google.Cloud.Retail.V2;
+using Google.LongRunning;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+/// <summary>
+/// The purge user event sample class.
+/// </summary>
+public class PurgeUserEventSample
+{
+    /// <summary>
+    /// The the user event.
+    /// </summary>
+    /// <returns>The usert event.</returns>defaultCatalog
+    public static UserEvent GetUserEvent()
```

Comment:
This method is not part of the sample code but of the tutorial only.
Consider removing it from within the region tags.

Reply:
Removed it from **Purge** and **Rejoin** samples. Now, the method **GerUserEvent()** exists only in **WriteUSerEvent** sample.